AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -2043,6 +2069,19 @@ public static Object objectForKeyPathComponents(Object[] pathComponents) {
     return messages;
   }
 
+  /**
+   * Set this to true if you want details about the variable assignments
```

Comment:
These comments don't seem to correspond to method which is a getter method and not a setter method for a flag?